### PR TITLE
[NRH-280] Social metatags

### PR DESCRIPTION
**Pull request by michael-caktus**
Before merging to develop:
- [x] Previous bugs fixed?
- [x] Release created?

#### What's this PR do?
Adds some social media metatag fields to revenue program in the admin.
Adds <meta> tags for OG and twitter crawlers. Uses rev program values or defaults.

#### How should this be manually tested?
You can inspect the head of the donation page for meta tags. Or you can visit the [facebook share debugger](https://developers.facebook.com/tools/debug/) or the [twitter card validator](https://cards-dev.twitter.com/validator).

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No

### CODE REVIEW

File: spa/src/components/donationPage/DonationPageRouter.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ function DonationPageRouter() {
       { method: 'GET', url: ORG_STRIPE_ACCOUNT_ID, params: requestParams },
       {
         onSuccess: ({ data: responseData }) => {
+          console.log('that org stripe id: ', responseData);
```
Comment (JeremyGibson) [inline spa/src/components/donationPage/DonationPageRouter.js:74]: extra console logs

File: apps/common/models.py [hunk 2]
```diff
@@ -24,3 +25,19 @@ def __str__(self):
...
         postal_code = " " + self.postal_code if self.postal_code else ""
         return f"{self.address1}{address2}, {self.city}, {self.state}{postal_code}"
+
+
+class SocialMeta(models.Model):
+    # opengraph truncates titles over 95chars
+    title = models.CharField(max_length=95, blank=True)
+    description = models.TextField(blank=True)
+    url = models.URLField(blank=True)
+    card = SorlImageField(null=True, blank=True)
```
Comment (JeremyGibson) [inline apps/common/models.py:35]: From the AC it looks like this is missing a twitter handle

**Review by JeremyGibson — CHANGES_REQUESTED** [review]
Looks to be missing configurable twitter handle.

File: apps/common/models.py (hunk 2 above)
Comment (michael-caktus) [reply apps/common/models.py:35]: `twitter_handle` was added to RevProgram in a previous ticket.

**Review by JeremyGibson — APPROVED** [review]

**Pull request merged**